_layouts/default.html

```diff
@@ -37,7 +37,7 @@
       <h2>Privacy information</h2>
       <p>We are managing our own analytics using Matomo.
       <a href="https://matomo.org/faq/general/faq_18254/" target="_blank" rel="noopener">Know what it collects</a>.</p>
-      <a href="javascript:matomoOptOut()" class="btn">Reject</a>
+      <button onclick="matomoOptOut();closePrivacyBanner()" class="btn">Reject</button>
```

Comment:
why not, but that's inconsistent with the accept button

Reply:
yes but that is not a problem  
I think using two different tags it better than use a javascript function in an anchor tag ;)
but I can make the other link a button if you want tho

Reply:
Might want to use the `js-privacy-banner-close` on the two, and adapt the js (use querySelector**All** with a loop

Reply:
And add a `js-matomo-opt-out` class too

Reply:
what?

Reply:
Instead of using onClick which is some IE9 level stuff you use the `js-privacy-banner-close` on the two, iterating through the two buttons with querySelectorAll (querySelector returns a single node while querySelectorAll returns a list of nodes)

Reply:
but in which js file? also I could make a loop with document.getElementByClass() instead of using queries selectors

Reply:
There's no js files, only script tags.

Reply:
OK